src/tictactoe/assets/js/components/Board.js

```diff
@@ -1,5 +1,27 @@
 import * as React from 'react'
+import { GameRoundState } from '../actions'
 
-const Board = () => <div><h2>Board will be here</h2></div>
+const Board = ({ gameState }) => {
+    const isPlayersTurn = gameState == GameRoundState.PLAYER_MOVES
```

Comment:
This is an interesting one. Because, first of all, I still have doubts with naming there - it defines the usage.
At the moment `GameRound` is actually this extensible container which specifies the properties of the round. While `GameRound.gameState` is just an enum reflecting current phase of the game.
I'm avoiding to use broad `Game` name, because in the future a big Game/Session/Match could consist of several `GameRounds` in a row.

Need to align usage of terms: `Game`,`GameRound`, `Game(Round)State`